### PR TITLE
MD Tutorial

### PR DESCRIPTION
**Pull request by markverb1**
**Resolves**

Some people who don't know MD can't make tutorials because they don't know it. This makes a tutorial for people like that.

**Changes**

Created the MD tutorial.

**Reason for changes**

So people who don't know MD can learn it.

**Tests**

Tested on [Replit](replit.com).

### CODE REVIEW

**Comment by TheColaber** [comment]
Should we make this a pinned tutorial instead of being a page? I think it could be hardcoded... we could remove that code in index.js where it removes all the storage.

File: pages/mdtutorial/index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Home - When Flag Clicked</title>
+    <meta property="og:title" content="When Flag Clicked" />
```
Comment (TheColaber) [inline pages/mdtutorial/index.html:5]: Fix this

File: pages/mdtutorial/index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      window.app = Vue.createApp({
+        data() {
+          return {
+            html: marked(
+              `
+<center><h1>When Flag Clicked Markdown Tutorial</h1></center>
+<center><h2>This tutorial will teach you the basics of markdown.</h2></center>
+
+<div style="margin: auto; width: 50%;">
+<h2>Headings</h2>
```
Comment (TheColaber) [inline pages/mdtutorial/index.html:28]: TODO: convert to MD?

**Review by TheColaber — CHANGES_REQUESTED** [review]

File: pages/mdtutorial/index.html (hunk 1 above)
Comment (markverb1) [reply pages/mdtutorial/index.html:5]: the space at the end?

Comment (TheColaber) [reply pages/mdtutorial/index.html:5]: No, fix it so the title it not "Home" and also change the og:title too

File: pages/mdtutorial/index.html (hunk 2 above)
Comment (TheColaber) [reply pages/mdtutorial/index.html]: Problems happened when converting... please fix

**Review by TheColaber — APPROVED** [review]

**Comment by markverb1** [comment]
Might actually make some buttons at the top of the editor which insert markdown (just like Scratch and GH) so this might become useless.

**Pull request merged**